Server/JokesOnYou.Web.Api/Services/Interfaces/IUserService.cs

```diff
@@ -13,5 +15,7 @@ public interface IUserService
         Task<User> GetUserById(string id);
```

Comment:
when do we return a complete user to the view? (user contains password etc...)

Reply:
changed to plain Task, so register only creates user.